primers/pcr/example_test.go

```diff
@@ -0,0 +1,64 @@
+package pcr_test
+
+import (
+	"fmt"
+	"github.com/TimothyStiles/poly/primers/pcr"
+)
+
+// This example shows how to design a sequence.
+func Example_basic() {
+	// We aim to amplify the tfoX gene.
```

Comment:
what is the tFox gene? wiki reference would do.

Reply:
I don't think that is relevant, so I removed all references to tfoX so that people aren't curious.